scheduler/src/cook/kubernetes/api.clj

```diff
@@ -416,9 +443,15 @@
                                                                  .getResources
                                                                  .getRequests
                                                                  convert-resource-map))
-                                                           containers)]
-                               (apply merge-with + container-requests))))
-                      (apply merge-with +)))
+                                                           containers)
+                                   {:keys [gpus] :as resource-map} (apply merge-with + container-requests)
+                                   gpu-model (-> pod .getSpec .getNodeSelector (get "cloud.google.com/gke-accelerator"))]
+                               (if (and gpu-model (pos? gpus))
```

Comment:
This (if.... ) clause has been used in two spots and can be refactored out.

Reply:
refactored into a function `add-gpu-model-to-resource-map`. open to other naming suggestions!